src/ui/React/LogBoxManager.tsx

```diff
@@ -260,35 +263,34 @@ function LogWindow(props: IProps): React.ReactElement {
             }}
           >
             <Box className="drag" display="flex" alignItems="center" ref={draggableRef}>
-              <Typography color="primary" variant="h6" title={title(true)}>
+              <Typography color="primary" variant="h6" sx={{ marginRight: "auto" }} title={title(true)}>
                 {title()}
               </Typography>
 
-              <Box position="absolute" right={0}>
-                {!workerScripts.has(script.pid) && (
-                  <Button onClick={run} onTouchEnd={run}>
-                    Run
-                  </Button>
-                )}
-                {workerScripts.has(script.pid) && (
-                  <Button onClick={kill} onTouchEnd={kill}>
-                    Kill
-                  </Button>
-                )}
-                <Button onClick={minimize} onTouchEnd={minimize}>
-                  {minimized ? "\u{1F5D6}" : "\u{1F5D5}"}
+              {!workerScripts.has(script.pid) && (
```

Comment:
nit: could || these two buttons based on the condition

Reply:
Yeah no need to do the same check twice. Changed it to a ternary.